### DISCUSSION TITLE
Companies / Sites using SWC

### DISCUSSION THREAD

**Discussion by kdy1** [swc-project/swc#9695] [RFC]
If you are using SWC directly or indirectly from your company or website, please leave a comment here!
I'll add them to https://swc.rs/docs/customers

**Comment by Brooooooklyn** [comment]
https://github.com/toeverything/AFFiNE
https://affine.pro

**Reply by kdy1** [reply]
I added to https://swc.rs/docs/customers! Thank you!

**Comment by mayur-dit** [comment]
https://apimaker.dev

**Reply by kdy1** [reply]
I added to https://swc.rs/docs/customers! Thank you!

**Comment by stovmascript** [comment]
https://mediaboard.com/

**Reply by kdy1** [reply]
Thanks! I added to https://swc.rs/docs/customers

**Comment by jantimon** [comment]
Galaxus/Digitec (a European online shop with $3bn revenue) - we use it for i18n and css

https://www.galaxus.ch/en
https://www.galaxus.de/en
https://www.galaxus.fr/en
https://www.galaxus.at/en
https://www.galaxus.nl/en
https://www.galaxus.be/en
https://www.digitec.ch/en

**Reply by kdy1** [reply]
Thanks! I added to https://swc.rs/docs/customers

**Comment by hamlim** [comment]
Added it to both [Wayfair](https://wayfair.com) and [Fireworks.ai](https://fireworks.ai) codebases!

**Reply by kdy1** [reply]
Thanks! I added to https://swc.rs/docs/customers

**Comment by araujobarret** [comment]
We added it to our web app https://artsy.net and we're trying to incorporate in our app built with React Native to speed up tests but we're facing some bumps with relay plugin setup, I think soon we'll have it there too.

**Reply by kdy1** [reply]
I'm trying to add artsy.net to https://swc.rs/docs/customers but the SVG logo does not render properly for an unknown reason. Can you upload a copy of the logo SVG file?

**Comment by chriskjaer** [comment]
We use it at https://landfolk.com. We've also created our own internal swc-plugin for a custom tailwind syntax we use (which we hope to open source soon 🤞)

**Reply by kdy1** [reply]
Thanks! I added to https://swc.rs/docs/customers .

**Comment by ryanelian** [comment]
We use it in [HENNGE](https://hennge.com/jp/about/brand/) through [Vite plugin React SWC](https://github.com/vitejs/vite-plugin-react-swc).

We migrated our codebase from normal React Router to a React Router meta-framework developed in-house capable of File-Based Routing, Automatic Per-Route Code-Splitting, and other features similar to Remix framework.

**Reply by kdy1** [reply]
Thanks! I added to https://swc.rs/docs/customers .

**Comment by csrujanreddy** [comment]
We transitioned from Babel to SWC, and our builds are now 400x faster! 😊

[themediaant.com](https://www.themediaant.com)
[div.foo](https://div.foo)

**Reply by kdy1** [reply]
Added via https://github.com/swc-project/website/commit/4a40cd3493ede625ffa4eccbad6184f1a6cae62c ! Thank you!
